marketplacetests/marketplace/app.py

```diff
@@ -104,6 +104,7 @@ def search(self, term):
 
     def set_region(self, region):
         # go to the :debug page
+        self.wait_for_element_displayed(*self._search_locator)
```

Comment:
Thanks @davehunt. I also updated the `search` method to wait for visibility as well. r? again please